### ISSUE TITLE
please update the document

### ISSUE THREAD

**Issue by malikasinger1** [api-ai/apiai-nodejs-client#77]
please update the document, why document is incomplete?

**Comment by she-dev** [comment]
Hi @malikasinger1, you mean the installation and usage documentation, right?

Were you able to run the client and have an idea of what could be added? If so, I'd be happy to discuss it and help making the docs clearer as I'm arriving on this project now and would like to try it.

**Comment by malikasinger1** [comment]
yes i'm able to run but there is lack of example and geting started document

**Comment by malikasinger1** [comment]
issue #77
